pages/docs/config/models.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Models
+date: "2021-06-22"
+pullQuote: "Learn how to configure apps using Grouparoo's UI Config."
+---
+
+Models are how you define the shape you want your records to have in Grouparoo. Once you've created a model, you can add sources and properties to that model to generate different record types. For example, you may want one model that defines leads, another for customers, and another for site visitors. [See this doc](/docs/getting-started/product-concepts) to learn more about Grouparoo's core concepts.
+
+With [UI Config](/docs/config) running and registered, follow the steps below to configure your model(s).
+
+### Add a Model
```

Comment:
The other pages have a "Step 1"

Reply:
It's so awkward without a "Step 2"!  I'll add it for consistency.